snapcraft/project/_arch_mapping.py

```diff
@@ -0,0 +1,142 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017-2018 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import platform
+import sys
+
+from snapcraft.internal import errors
+
+
+_ARCH_TRANSLATIONS = {
```

Comment:
At least according to how I interpret the underscores (happy to be corrected), an underscore here indicates that this variable is intended to be private to this module. However, we directly import it into other modules. That means it's not actually private. Consider either renaming this, or reworking things to properly encapsulate it.